### DISCUSSION TITLE
How to set working area in LaserGRBL

### DISCUSSION THREAD

**Discussion by NewtoLASER** [arkypita/LaserGRBL#2400] [Q&A]
Hello,

I have an annoying problem, I bought a Vevor CNC S3018 last year and have now brought it back from under the "dust' because i have extra time.
Everything works...only when I want to laser something it is small and not as on the screen (zoomed in) what i see...
can anyone help me, my work surface (bed) is 300mmx180mm.
I dont know to where to enter this data into Lasergrbl?
![Schermafbeelding 2024-11-04 001743](https://github.com/user-attachments/assets/2890e1ad-90a6-4135-a5ba-2638591f0425)
![Schermafbeelding 2024-11-04 003702](https://github.com/user-attachments/assets/5d247b2f-1cc6-47e1-93ea-da11b9eb759f)
![Schermafbeelding 2024-11-04 003727](https://github.com/user-attachments/assets/a976cca1-4717-404c-9518-d8efeaf75287)
![Schermafbeelding 2024-11-04 003911](https://github.com/user-attachments/assets/facdcba3-1f24-4a6f-8f84-65825617c84c)

**Comment by Tribo-2dot0** [comment]
Na primeira imagem você deve atualizar o tamanho da sua área de trabalho (300x180) em $130 e $131, na imagem está 200x200.
Na terceira imagem é onde você insere o tamanho da gravação em milímetros (na imagem está 20,4x20,7mm)

**Reply by NewtoLASER** [reply]
Hello ,
I have tried this settings but even its stills laser very small , its like its laser not bigger as 1cm while i want it to laser on the full work area
(bed).

**Reply by Tribo-2dot0** [reply]
$100 e $101 estão ajustados em 53, não é um valor usual, o mais comum é 80 ou 100.
Digite $RST=$ no console para restaurar as configurações originais, ajuste os valores de $130 e $131 para o tamanho da sua área de trabalho, faça $32=1, se necessário...

**Reply by NewtoLASER** [reply]
i have tried other settings but its hard to get a good work that he laser , I have order this for laser/cnc engraving....but to much controls for each laser object /cnc....not properly going to home , set 0 , always adjusting manuel on the x , y as.... sometimes out of the working area and the motors making noises so i need to stop the power directly in the program..... anyway thanks for the feedback but for me after few days trying , research for the cnc s3018 but hard to get correct information .
for me it didn't work good , maybe i missing something a step i dont know. i will rest it for now at the moment until i want it to using it again in the future
